src/donationswap.py

```diff
@@ -625,7 +631,7 @@ def _get_match_score(self, offer_a, offer_b, db):
 		score = round(score, 4)
 		return score, reason
 
-	def getActualAmounts(self, my_offer, their_offer):
+	def _get_actual_amounts(self, my_offer, their_offer):
```

Comment:
never mind, I've renamed it, @Athight . Your time is more valuable, because you're the only one who can implement gift aid.

Reply:
😲 i am the chosen one?
I'm sure you'd get it if you needed to, but cheers 😜